modules/rosa-installing-prerequisites.adoc

```diff
@@ -12,7 +12,7 @@ Complete the following prerequisites before creating your {product-title} cluste
 
 . Select the AWS account you want to use.
 +
-Unless you are just testing out {product-title}, it is recommended to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
+Unless you are just testing out {product-title}, the best practice is to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
```

Comment:
The ISG recommends not using "best practice". It has come up for me a few times, so I just wanted to mention it here. :)